### ISSUE TITLE
Add more open questions at end?

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#79]
I would suggest, you could ask how people configure their applications, what kind of configuration they prefer, why they prefer it, etc.

**Comment by markus2330** [comment]
Yes, I also think we cannot apply these suggestions directly. Nevertheless we could ask some concrete open questions. I am thinking about:

- Describe the limitations of current configuration systems you face.
- Describe the ideal configuration system for your operating system and applications.

What do you think?

**Comment by Namoshek** [comment]
Such a question could be interesting as open question to check on previous answers (of course some interpretation would be required). We could add this to the feedback page and give a good hint that it is optional, but if they would like to share their frustration/ideas (what they like/don't like concerning current systems) with us, they are welcome.

**Comment by omnidan** [comment]
I think it would be interesting too (as long as we clearly state it's optional). I don't think many people will answer, but the ones that do could be very useful feedback (e.g. someone being very angry at a certain system will be more likely to write something there to vent, which can be useful information for us).

**Comment by sirblackheart** [comment]
Why not add them as suggestion to the `Further comments?` textbox.

**Comment by markus2330** [comment]
If you think further two boxes would be overwhelming, its best to add these two questions to the one box we have.

Are there further suggestions for questions?

**Comment by Namoshek** [comment]
Personally I wouldn't mix "feedback to survey" (which seems to be a meta-question) and topic related questions. But I would only provide one additional text box for "everything else they want to share with us".

**Comment by markus2330** [comment]
Ok, sounds good.

**Comment by Namoshek** [comment]
I added another feedback question just before the survey-feedback question. Please review.

> We are also interested in learning about limitations you currently face with your configuration system(s) and what the ideal configuration system would be for you.

> If there is anything else you want to share with us regarding configuration systems, you can also write it down here!

**Comment by markus2330** [comment]
Thank you. Btw. in the meeting I asked the question and we agreed that is unlikely that someone thought about that.

Do you have additional (provoking) subquestions that more people thought about?

E.g. configuration as a code vs. simple configuration files or something like that?
